### PR TITLE
Fix all_success? method for use in the ci:run task

### PR DESCRIPTION
**Pull request by jleeman-mdsol**

### CODE REVIEW

File: lib/kender/command.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def run
     class << self
 
       def all_success?
-        all.inject(true) {|all_result, command_result| all_result && command_result }
+        all.inject(true) {|all_result, command_result| all_result && command_result.success }
```
Comment (ykitamura-mdsol) [inline lib/kender/command.rb:27]: how about 
```suggestion
        all.all?(&:success)
```
?

**Review by ykitamura-mdsol — COMMENTED** [review]
good catch!

**Comment by jcarres-mdsol** [comment]
We would need to update the changelog and the version also

File: lib/kender/command.rb (hunk 1 above)
Comment (jleeman-mdsol) [reply lib/kender/command.rb]: https://github.com/mdsol/kender/pull/79/commits/8036dd15726b49b43dde0db9276e4ae958b793aa

**Comment by jleeman-mdsol** [comment]
Thanks guys, added Yohei's much cleaner fix, and CHANGELOG/version change requested by Jordi.

**Review by jcarres-mdsol — APPROVED** [review]

**Pull request merged**